cmd/dep/init.go

```diff
@@ -258,6 +262,24 @@ func getProjectData(ctx *dep.Ctx, pkgT gps.PackageTree, cpr string, sm *gps.Sour
 				return projectData{}, errors.Wrap(err, "sm.DeduceProjectRoot") // TODO: Skip and report ?
 			}
 
+			start := time.Now()
+			message := make(chan string, len(v.P.Imports))
+			go func() {
+				goStart := time.Now()
+				defer fmt.Fprintf(os.Stderr, "Go Duration: %s\n", time.Since(goStart))
+				err2 := sm.SyncSourceFor(gps.ProjectIdentifier{ProjectRoot: pr})
+				fmt.Fprintf(os.Stderr, "SyncSourceFor Duration: %s\n", time.Since(goStart))
+				if err2 != nil {
+					message <- fmt.Sprintf("Unable to clone: %s\n", pr)
+				}
+				message <- fmt.Sprintf("Cloning: %s", pr)
+			}()
+
+			select {
+			case m := <-message:
```

Comment:
I'm not sure it's necessary to print results from the main goroutine. Doing so ends up serializing all the network calls, because the loop can't move on to the next local dep to scan until the syncSourceFor call completes.

Maybe instead, make a closure at the beginning of `getProjectData()` that we can invoke in a goroutine every time we encounter a new project root.

Of course, we'll need to take some care there to deal with printing from multiple goroutines - if nothing else, it might be better to print only either before or after `SyncSourceFor` completes.